src/sql/base/browser/ui/breadcrumb/breadcrumb.component.ts

```diff
@@ -30,7 +30,7 @@ import { IDisposable } from 'vs/base/common/lifecycle';
 				`
 })
 export class BreadcrumbComponent implements OnInit, OnDestroy {
-	private menuItems: MenuItem[] = [];
+	protected menuItems: MenuItem[] = [];
```

Comment:
why is this necessary?

Reply:
its the cleaniest way to defined a variable that isn't being used by the class directly but is being used in the html template. Since the tslinter can't parse angular code it will report this as being unused and throw a type error, even if it is being used in the template. The other solution is to type ignore it (which is a valid approach) or make it public (which I don't think is the right approach)

Reply:
More I think about it, the more I think it might be cleaner here to just add a tslint disable tag and add a comment that says that the variable is used in the angular template.

Reply:
nvm this can't be fixed via tslint rules since this is a compiler feature not a tslint feature